tests/fxl-layout_duplication/OPS/package.opf

```diff
@@ -0,0 +1,36 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<package xmlns="http://www.idpf.org/2007/opf" xmlns:epub="http://www.idpf.org/2007/ops" version="3.0" xml:lang="en" unique-identifier="pub-id">
+  <metadata xmlns:dc="http://purl.org/dc/elements/1.1/">
+    <dc:coverage>Fixed Layout</dc:coverage>
+    <dc:creator>Wendy Reid</dc:creator>
+    <dc:date>2021-11-24</dc:date>
+    <dc:description>
+      When rendition:layout is duplicated, EPUBCheck must return an error.
+    </dc:description>
+    <dc:identifier id="pub-id">fxl-layout_duplication</dc:identifier>
+    <dc:language>en</dc:language>
+    <dc:title>rendition:layout duplication</dc:title>
+    <link rel="dcterms:rights" href="https://www.w3.org/Consortium/Legal/2015/copyright-software-and-document"/>
+    <link rel="dcterms:rightsHolder" href="https://www.w3.org"/>
+    <meta property="dcterms:isReferencedBy">https://www.w3.org/TR/epub-33/#layout-duplication</meta>
```

Comment:
This anchor does not exist, I presume this should be the subject of a separate PR on the content document. However, I believe the current convention is to use the same filename of the test as an anchor (@dlazin should be the judge of this), ie, the anchor should probably be `fxl-layout_duplication`.

(This may be a repeated pattern over all the 12 tests...)

Reply:
Also, but that may be a more generic issue to be raised separately: there is no statement in the RS on what an RS should do if it hits a duplicated `rendition-layout`: ignore both, pick the first one, pick the second one?

Actually, anticipating a change in the RS test, it may be a better idea to put a `pre-paginated` _and_ a `reflowable` setting